frontend/app/views/spree/shared/_translations.html.erb

```diff
@@ -1,11 +1,20 @@
 <script>
   window.addEventListener('turbolinks:load', function() {
-      Spree.translations = <%==
-    {
-      coupon_code_applied: Spree.t(:coupon_code_applied),
-      coupon_code_removed: Spree.t(:coupon_code_removed),
-      coupon_code_error_icon: image_path('error.svg')
-    }.to_json
-  %>
+    Spree.translations = <%==
+      {
+        coupon_code_applied: Spree.t(:coupon_code_applied),
+        coupon_code_removed: Spree.t(:coupon_code_removed),
+        coupon_code_error_icon: image_path('error.svg'),
+        state: Spree.t(:state)
+      }.to_json
+    %>
+  });
+
+  window.addEventListener('DOMContentLoaded', function() {
+    Spree.translations = <%==
+      {
+        state: Spree.t(:state)
```

Comment:
duplicate key, also we don't need an additional block, if it's required put code inside frontend/app/assets/javascripts/spree/frontend/checkout/address.js in `window.addEventListener`

Reply:
If I try to add this code to address.js I have two problems:
1. When should the `window.addEventListener` be called? I've tried on `load`, `DOMContentLoaded`, `turbolinks:load` and none of it works.
2. [ts] issue with `Spree.t(:state)`, because it raises `Argument expression is expected`